packages/ng/callout/src/lib/callout/callout.component.ts

```diff
@@ -0,0 +1,72 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { Palette, SizeMS } from '@lucca-front/ng/core';
+
+@Component({
+	selector: 'lu-callout',
+	standalone: true,
+	imports: [CommonModule],
+	templateUrl: './callout.component.html',
+	styleUrls: ['./callout.component.scss'],
+	changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class CalloutComponent {
+	@Input()
+	/**
+	 * The title of the callout
+	 */
+	title: string;
+
+	@Input()
+	/**
+	 * Which palette should be used for the entire callout.
+	 * Defaults to none (inherits parent palette)
+	 */
+	palette: Palette = 'none';
+
+	@Input()
+	/**
+	 * Which size should the callout be? Defaults to medium
+	 */
+	size: SizeMS = 'm';
```

Comment:
I am a bit afraid of the future of this kind of type. We can have up to 9 sizes (XXXS, XXS, XS, S, M, L, XL, XXL, XXXL) but not all components can use all sizes. I would keep it simple with a `'m' | 's'` inlined here